Source/OrchBase/Composer.cpp

```diff
@@ -914,6 +1502,10 @@ int Composer::generateSupervisor(ComposerGraphI_t* builderGraphI)
         supervisor_h << "#define SUPSTATE(a)         supervisorState->a\n";
         supervisor_h << "#define MSG(a)              message->a\n";
         supervisor_h << "#define PKT(a)              message->a\n";
+        supervisor_h << "#define REPLY(a)            reply->a\n";
+        supervisor_h << "#define BCAST(a)            bcast->a\n";
+        supervisor_h << "#define RTSREPLY()          __rtsReply=true\n";
+        supervisor_h << "#define RTSBCAST()          __rtsBcast=true\n";
         supervisor_h << "\n";
```

Comment:
Looking back at my notes, these were put here because they were needed in the generated header, which does not #include any of the other Supervisor headers (there are reasons).

Looking at why it is needed in the generated header, it is because the "code" section is placed in the header. This is placed in the header so that any types from includes in the code may be used in the properties/state. 

This is probably worth having a chat about.

Reply:
Yes, let's have a natter.

Reply:
Will move to  `SupervisorMacros.h` or equivalent.

Reply:
I have ended up moving these to the Supervisor header and addressing the reasons why I did not do this before.